CONTRIBUTING.md

```diff
@@ -0,0 +1,125 @@
+
+Create a folder say AzureWebJobsStorage
+
+`npm install azurite@2.7.1 -g`
+
+`azurite -l ./AzureWebJobsStorage`
+
+```
+ _______                   _             
+(_______)                 (_)  _         
+ _______ _____ _   _  ____ _ _| |_ _____ 
+|  ___  (___  ) | | |/ ___) (_   _) ___ |
+| |   | |/ __/| |_| | |   | | | |_| ____|
+|_|   |_(_____)____/|_|   |_| \__)_____)
+                                         
+Azurite, Version 2.7.1
+A lightweight server clone of Azure Storage
+
+Azure Table Storage Emulator listening on port 10002
+Azure Queue Storage Emulator listening on port 10001
+Azure Blob Storage Emulator listening on port 10000
+```
+
+### Visual Studio Code Extensions
+
+The following extensions should be installed if using Visual Studio Code for debugging:
+
+- C# for Visual Studio Code (powered by OmniSharp)
+- Azure Functions Extensions for Visual Studio Code v0.19.1 and above.
+
+### Setting up durable-js debugging
+
+- Create a Durable Functions Orchestrator for FunctionChaining pattern using [starter templates](https://docs.microsoft.com/en-us/azure/azure-functions/durable/quickstart-js-vscode)
+  Note: In this starter template, ignore the line that says: "On a Mac or Linux computer, you must set the AzureWebJobsStorage property to the connection string of an existing Azure storage account". We will be setting up the AzureWebJobsStorage property to `UseDevelopmentStorage=true`
+
+- In host.json, remove the extensionsBundle portion to enable specific version debugging. Provide a hub name (else remove the entire extensions portion to default to DurableFunctionsHub) Here's how the host.json should look like:
+
+```
+{
+  "version": "2.0",
+  "extensions": {
+    "durableTask": {
+      "hubName": "{hubName}"
+    }
+  }
+}
+```
+
+- `func extensions install` (this will install an extensions.csproj that contains the version of DurableTask as seen below)
+
+```xml <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.DurableTask" Version="1.8.2" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Script.ExtensionsMetadataGenerator" Version="1.1.0" />
+  </ItemGroup>
+```
+
+### Setting up Durable Extension .NET debugging
+
+- Download the source code of DurableTask extension from [here](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.DurableTask)
+- In launch.json, add requireExactSource and justMyCode fields for .NET Core Attach task.
+
+```
+ {
+            "name": ".NET Core Attach",
+            "type": "coreclr",
+            "request": "attach",
+            "processId": "${command:pickProcess}",
+            "requireExactSource": false,
+            "justMyCode": false
+ }
+ ```
+
+### Debugging end-to-end
+
+1. Open the Azure Storage Explorer and connect to the local storage emulator or the storage account you are using.
```

Comment:
Should we add Azure Storage Explorer (with a download link) to the prereqs list above?

Reply:
There's already a link to it in the "Tools" section. I have updated the link to point to the download URL though